.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'd prefer to enable `black` the same way as `pyre`. If we enable `black` then we also need to make sure out jdwp github actions run pyre check and fail if code is not formatted properly.

I would make this kind of change in separate PR.